### PR TITLE
I solved HW4 condition Operators task4 Stock

### PR DESCRIPTION
**Pull request by avp02**

### CODE REVIEW

File: src/main/java/com/avp/homework4_conditionOperators/task4_stock/service/StockService.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        stock.setCurrentPrice(newPrice);
+        if (stock.getMinPrice() > newPrice) {
+            stock.setMinPrice(newPrice);
+        } else if (stock.getMaxPrice() < newPrice) {
+            stock.setMaxPrice(newPrice);
+        }
+
+    }
+
+    public String getPriceInformation(Stock stock) {
```
Comment (kashlyak) [inline src/main/java/com/avp/homework4_conditionOperators/task4_stock/service/StockService.java:17]: Почитай про StringBuilder.

**Review by kashlyak — APPROVED** [review]

**Review by olgaZskiba — APPROVED** [review]
lgfm

File: src/main/java/com/avp/homework4_conditionOperators/task4_stock/service/StockService.java (hunk 1 above)
Comment (avp02) [reply src/main/java/com/avp/homework4_conditionOperators/task4_stock/service/StockService.java:17]: I understood

**Review by akupreychik — APPROVED** [review]
LGFM

**Pull request merged**